contrib/upstart/cilium-net-daemon.conf

```diff
@@ -1,2 +1,2 @@
 env PATH=/usr/local/clang/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/sbin:/sbin:/bin
-exec cilium -D daemon run -d eth1
+exec cilium -D daemon run --ipv4 --lb
```

Comment:
Yes, if k8s is enabled, services should always be handled.

The maps can be created unconditionally. As for whether to clear them at startup, we can make use of the `--keep-config` flag to opt out of this behavior.